core/src/test/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectRegistrationRepositorySpec.scala

```diff
@@ -56,4 +56,19 @@ class GoogleProjectRegistrationRepositorySpec extends AnyFlatSpec with TestDrive
     }
   }
 
+  behavior of "deleteGoogleProjectRegistration"
+
+  it should "do nothing if trying to delete a record that does not exist" in withDefaultTestDatabase {
```

Comment:
Can we add a test that confirms it gets deleted as well?

Reply:
I've added one in the component, which does the actual deleting, then one here to make sure it calls the component, not sure that makes the most sense though